app/assets/stylesheets/detailsPage.scss

```diff
@@ -80,6 +122,20 @@ h1 {
   text-align: center;
 }
 
+.tagHeader {
+  color: $gray-color;
+  font-weight: normal;
+  font-size: 12pt;
```

Comment:
I'm pretty sure these font-size and font-weight values are the default? Are they necessary?

Reply:
Font wise, I find it's always wise to spell it out, because defaults vary between browsers